dnf/yum/rpmtrans.py

```diff
@@ -129,15 +130,19 @@ def __init__(self):
                        self.PKG_OBSOLETE: _('Obsoleting'),
                        self.PKG_REINSTALL: _('Reinstalling'),
                        self.PKG_UPGRADE: _('Upgrading'),
-                       self.PKG_VERIFY: _('Verifying')}
+                       self.PKG_VERIFY: _('Verifying'),
+                       self.TRANS_PREPARATION: _('Preparing'),
+                       self.PKG_SCRIPTLET: _('Running scriptlet')}
         self.fileaction = {self.PKG_CLEANUP: 'Cleanup',
                            self.PKG_DOWNGRADE: 'Downgraded',
                            self.PKG_ERASE: 'Erased',
                            self.PKG_INSTALL: 'Installed',
                            self.PKG_OBSOLETE: 'Obsoleted',
                            self.PKG_REINSTALL: 'Reinstalled',
                            self.PKG_UPGRADE:  'Upgraded',
-                           self.PKG_VERIFY: 'Verified'}
+                           self.PKG_VERIFY: 'Verified',
+                           self.TRANS_PREPARATION: _('Preparing'),
```

Comment:
@ignatenkobrain Fixed, if anything else is blocker to merge it, please let me know.